R/Weather.R

```diff
@@ -1,26 +1,7 @@
 #A function to summarize hourly weather data
 #with options to summarize by day (level=daily) or month (level=monthly)
 
-#library(dtplyr)
-library(dplyr)
-library(tidyr)
-library(lubridate)
-
-#' Return normalized path for all operating systems
-#'
-#' @param ReferencePath a path to join with current working directory
-#' @param BasePath Current working directory else path given
-#'
-#' @return
-#' @export
-#' @examples
-#' FullPath('PortalData/Rodents/Portal_rodent.csv')
-#' FullPath('PortalData/Rodents/Portal_rodent.csv', '~')
-FullPath <- function( ReferencePath, BasePath=getwd()){
```

Comment:
Never mind, it's coming from `GetData`, sorry. I have to admit I still don't fully understand how to think about namespacing within packages in R.